drivers/counter/counter_nrfx_rtc.c

```diff
@@ -532,7 +534,7 @@ static int init_rtc(struct device *dev, u32_t prescaler)
 	NRF_RTC_Type *rtc = nrfx_config->rtc;
 	int err;
 
-	clock = device_get_binding(DT_INST_0_NORDIC_NRF_CLOCK_LABEL);
+	clock = device_get_binding(DT_INST_LABEL(0));
```

Comment:
Use of `DT_DRV_COMPAT` in this context is misleading, since this driver is for "nordic,nrf-rtc" compatible devices and here it only needs to access the clock device.
```suggestion
	clock = device_get_binding(DT_PROP(DT_INST(0, nordic_nrf_clock), label));
```